package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@hapi/good": "8.2.1",
     "@hapi/hapi": "18.3.2",
+    "@mojaloop/central-services-shared": "7.4.3",
```

Comment:
please update all dependencies and version of the package.json

Reply:
Change request performed.